.eslintrc

```diff
@@ -415,7 +415,8 @@
     "@typescript-eslint/no-unused-vars": [
       "error",
       {
-        "args": "none"
+        "args": "none",
+        "varsIgnorePattern": "_"
```

Comment:
Won't this ignore any variable with an underscore? According to https://eslint.org/docs/latest/rules/no-unused-vars#varsignorepattern (which is referenced by the typescript-eslint docs at https://typescript-eslint.io/rules/no-unused-vars/), the linter doesn't appear to require a full regex match

Reply:
I have updated this part. @U8NWXD PTAL